src/components/profile/Profile.js

```diff
@@ -150,18 +150,20 @@ export function Profile({ match }) {
     }, []);
 
     useEffect(() => {
-        if (account.balance?.total) {
+        if (account.balance) {
             let id = Mixpanel.get_distinct_id()
             Mixpanel.identify(id)
             Mixpanel.people.set_once({create_date: new Date().toString(),})
             Mixpanel.people.set({
                 relogin_date: new Date().toString(),
                 enabled_2FA: account.has2fa,
-                [accountId]: formatNEAR(account.balance.total) 
+                [accountId+'total']: formatNEAR(account.balance.total), 
```

Comment:
how is this processed on mixpanel side? Seems like some delimited between `accountId` and subfields might be needed?

Reply:
yes, I add _ instead. Thanks